### PR TITLE
feat : checkout page

### PR DESCRIPTION
**Pull request by rohit8686**
Features : 

- Items present in cart
- price details
- delivery address

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
Code looks good
PR is ready to merge

File: src/frontend/pages/Checkout/Checkout.jsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+          <hr />
+          <div class="flex space-between pt-1">
+            <p>Price ({cartData.length} items)</p>
+            <p>&#8377; {totalPrice}</p>
+          </div>
+          <div class="flex space-between pt-1">
+            <p>Discount</p>
+            <p>- &#8377;{totalDiscount}</p>
+          </div>
+          <div class="flex space-between pb-1 pt-1">
```
Comment (RahulSingh9131) [inline src/frontend/pages/Checkout/Checkout.jsx]: You can change class to className here.

Comment (rohit8686) [reply src/frontend/pages/Checkout/Checkout.jsx]: yeah will change it thanks for pointing it out

**Review by RahulSingh9131 — APPROVED** [review]
perfect use of state variable from reducers, css is clean too.

**Pull request merged**